lib/backends/swagger-client.js

```diff
@@ -28,11 +28,17 @@ class SwaggerClient {
       options.qs || options.parameters,
       options.pathnameParameters
     )
+
     const operationId = options.pathItemObject[options.method.toLowerCase()].operationId
-    return this.client.execute({
-      operationId,
-      parameters
-    })
+
+    const operationReference = operationId ? {
+      operationId
+    } : {
+      pathName: options.pathname,
+      method: options.method
+    }
```

Comment:
According to https://github.com/swagger-api/swagger-js#tryitout-executor 
`pathName` and `method` can be provided in place of `operationId`

Possibly even preferred in this case?

> // If `pathName` and `method`, then those are used instead of operationId. This is useful if you're using this dynamically, as `pathName` + `method` are guarenteed to be unique.

Keeping old behaviour when possible seemed like a less intrusive change tho :)